jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -426,8 +427,8 @@ public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilde
                 ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
                 String badgeHtml = null;
                 if(errCount > 0){
-                    //TODO: set link URL to correct error view perspective path
-                    String linkToErrorView = "#Error View?processInstanceId=" + value.getProcessInstanceId();
+                    //TODO: link to specific process instance ID
+                    String linkToErrorView = "#" + PerspectiveIds.EXECUTION_ERRORS;
```

Comment:
@rbarriuso regarding JIRA, we should open JBPM jiras for these instead of using BPMSPL anyway.